_content/reference/verses-of-the-elder-nuns-selections_hallisey-charles.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Therigatha Verses of the Elder Nuns Selections"
+authors:
+  - hallisey-charles
+external_url: "https://www.bhikkhuni.net/wp-content/uploads/2018/07/therigatha.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1MR5KHc3Yp3dBsaURJLDVlEFLu93vt4_7/view?usp=sharing"
+course: tg
+tags:
+  - poetry
+  - pali-canon
+year: 2018
+month: jun
+pages: 24
+---
+
+A selection of poems from the Therīgāthā, translated by Charles Hallisey. These selcetions were part of a  2018 retreat given at Spirit Rock by Jetsunma Tenzin Palmo,
```

Comment:
This should go in `excerpts` as it's a selection from this book: https://www.hup.harvard.edu/books/9780674427730